core/package.json

```diff
@@ -26,7 +26,8 @@
   },
   "files": [
     "dist/",
-    "loader/"
+    "loader/",
+    "components"
```

Comment:
Should this also end with a "/" like dist and loader?

Reply:
yes!